From 69f61f5301673b3df07153b56371eeb1d6e6af37 Mon Sep 17 00:00:00 2001
From: Juergen Spitzmueller <spitz@lyx.org>
Date: Fri, 11 Apr 2025 14:18:39 +0200
Subject: [PATCH] Fix compilation with Qt 6.9

which gets confused if moc_*.cpp
is included within the lyx:: namespace

(cherry picked from commit 892046ef90dfc13ee6fa10792bd56dbcf1feb2b7)
---
 src/Compare.cpp            | 3 +--
 src/support/Systemcall.cpp | 6 ++----
 3 files changed, 5 insertions(+), 6 deletions(-)

diff --git a/src/Compare.cpp b/src/Compare.cpp
index 82236aa8de..2b759b96b7 100644
--- a/src/Compare.cpp
+++ b/src/Compare.cpp
@@ -885,7 +885,6 @@ void Compare::Impl::writeToDestBuffer(ParagraphList const & pars) const
                mergeParagraph(dest_buf_->params(), *dest_pars_, pit);
 }

+} // namespace lyx

 #include "moc_Compare.cpp"
-
-} // namespace lyx
diff --git a/src/support/Systemcall.cpp b/src/support/Systemcall.cpp
index 7c4eaabc97..0ce85823ba 100644
--- a/src/support/Systemcall.cpp
+++ b/src/support/Systemcall.cpp
@@ -706,10 +706,8 @@ void SystemcallPrivate::killProcess(QProcess * p)
        }
 }

-
+} // namespace support
+} // namespace lyx

 #include "moc_SystemcallPrivate.cpp"
 #endif
-
-} // namespace support
-} // namespace lyx
